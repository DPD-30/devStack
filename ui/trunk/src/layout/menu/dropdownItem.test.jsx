import React from 'react'
import { renderToString } from 'react-dom/server'
import { HashRouter as Router } from 'react-router-dom'

import LayoutMenuDropdownItem from './dropdownItem.jsx'

jest.unmock('./dropdownItem.jsx')


describe('', () => {
    it('LayoutMenuDropdownItem should match snapshot', () => {
        const comp = renderToString(
            <Router>
                <LayoutMenuDropdownItem name="mockname" url="/mock" label="Mock Link" />
            </Router>
        )

        expect(comp).toMatchSnapshot()
    })
})
