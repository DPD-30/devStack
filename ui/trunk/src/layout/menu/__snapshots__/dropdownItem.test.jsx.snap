// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutMenuDropdownItem should match snapshot 1`] = `
<a name="mockname"
   tabindex="0"
   role="menuitem"
   class="dropdown-item"
   href="#/mock"
>
  Mock Link
</a>
`;
